docs/001_develop/03_client-capabilities/008_interaction/button.mdx

```diff
@@ -0,0 +1,169 @@
+
+# Button
+
+Implementation of a [button](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/button) component.
+
+Use cases:
+* Submit forms
+* Trigger actions
+* Create interactive elements
+* Set options
+
+## Example
+
+<ButtonDemo />
+
+<Tabs defaultValue="genesis" values={[{ label: 'Genesis', value: 'genesis', }, { label: 'React', value: 'react', }, { label: 'Angular', value: 'angular', }]}>
+
+  <TabItem value="genesis">
+
+    Declaration
+    ```html
+    <rapid-button>Button</rapid-button>
+    ```
+
+    Usage
+    ```typescript
+    @customElement({
+      name: 'my-element',
+      template: html`
+      <rapid-button @click=${x => x.buttonClicked()}>
+        Button
+      </rapid-button>
+    `,
+    })
+    export class MyElement extends GenesisElement {
+      buttonClicked() {
+        alert('Button Clicked')
+      }
+    }
+    ```
+
+  </TabItem>
+  <TabItem value="react">
+
+    Declaration
+    ```html
+    <rapid-button></rapid-button>
+    ```
+    Usage
+    ```typescript
+    export function MyComponent() {
+      const handleButtonClicked = (event) => {
+        console.log('Button clicked')
+      };
+
+      return (
+        <rapid-button onClick={handleButtonClicked}></rapid-button>
+      );
+    }
+    ```
+  </TabItem>
+  <TabItem value="angular">
+
+    Declaration
+    ```html
+    <rapid-button></rapid-button>
+    ```
+
+    Usage
+    ```typescript
+    import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+    import { FormsModule } from '@angular/forms';
+    @Component({
+      selector: 'my-root',
+      template: `
+        <rapid-button
+          (click)="onButtonClicked($event)"
+        ></rapid-checkbox>
+      `,
+      styleUrls: ['./my.component.css'],
+      standalone: true,
+      schemas: [CUSTOM_ELEMENTS_SCHEMA],
+      imports: [FormsModule],
+    })
+    export class AppComponent {
+      onButtonClicked(event: Event) {
+        console.log('Button clicked');
+      }
+    }
+    ```
+  </TabItem>
+</Tabs>
+
+## API
+
+Property and attribute binding examples for Genesis Component syntax.
+
+| Attribute/Property | Type | Description |
+|---|---|---|
+|autofocus|`boolean`|The boolean value of whether the checkbox is checked. |
+|appearance|`string`|Configures the component so the user cannot alter the state via the web browser.|
+|disabled|`boolean`|Same functionality as `readonly`, and additionally adds visual opacity to the component.|
+|form|`string`|Associates this component to a form. Form id needs to be passed. If no Id informed, then it will be associated with the ancestor form.|
+|name|`string`|Defines the name of the element.|
+|type|`string`|Defines the mode of the button.|
+|value|`string`|Defines the value associated with name.|
+
+The type attribute can be:
+* button (default) - performs an action when clicked
+* reset - can influence a group of components associated with the same form.
+
+### Examples (closing tag omitted):
+
+Attributes:
```

Comment:
Properties and attributes should be in this format 

See
https://github.com/genesiscommunitysuccess/docs/pull/1853/files#diff-b69edbdc88e90870eeee016c48b131788c8f74fcda501b2fae84807575e101abR329